GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -251,11 +279,16 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
         NotificationCenter.default.removeObserver(self)
     }
     
-    fileprivate func didPick(validatedDocuments documents: [GiniVisionDocument]) {
+    fileprivate func didPick(validatedDocuments documents: [GiniVisionDocument],
+                             completion: DocumentPickerCompletion?) {
         if let delegate = delegate {
-            delegate.camera(self, didCaptureDocuments: documents)
+            delegate.camera(self, didCaptureDocuments: documents, completion: completion)
         } else if let firstDocument = documents.first {
+            completion?(nil, nil)
             successBlock?(firstDocument)
+            if documents.count > 1 {
+                // TODO: Show a warning in the logger
```

Comment:
We don't like TODOs in a finished PR ;)

Reply:
But I prefer to have this before releasing the new version and implemented it when I create the Logger. Otherwise I will forget about this line.